packages/domain-picker/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line wpcalypso/import-docblock
+import type { DomainSuggestions } from '@automattic/data-stores';
+
+export function mockDomainSuggestion(
+	domainName: string | undefined
+): DomainSuggestions.DomainSuggestion | undefined {
+	if ( ! domainName ) {
+		return undefined;
+	}
+
+	return {
```

Comment:
@razvanpapadopol I am not completely sure how all these values are used and if there is any danger is mocking these values like below. I still need to look into this.

Reply:
They are used when creating the domain product to add to the cart. See https://github.com/Automattic/wp-calypso/blob/trunk/packages/launch/src/hooks/use-on-launch.ts#L35-L43 

But in this case they shouldn't be used at all since we're planning to use this function to create DomainSuggestion objects just for the (free) subdomain.